avoid install.sh calling 'make' (and hard set GIT_EXIST)
author: Kenneth Hoste (HPC-UGent)
--- ED/build/install.sh.orig	2017-03-27 12:00:10.321077952 +0200
+++ ED/build/install.sh	2017-03-27 11:59:40.200888391 +0200
@@ -94,8 +94,8 @@
 esac
 
 # Tag executables with a git version and branch name if possible.
-GIT_EXIST=`git rev-parse --is-inside-work-tree`
-if [ ${GIT_EXIST} == "true" -a ${USE_GIT} ]
+GIT_EXIST=""
+if [ "${GIT_EXIST}" == "true" -a ${USE_GIT} ]
 then
    GIT_TAG=`git branch -v | awk '/\*/ {print "-" $2 "-" $3}'`
    GIT_TAG=`echo ${GIT_TAG} | tr -d '()/[]'`
@@ -122,7 +122,7 @@
 touch dependency.mk
 
 #----- Launch the compiler. ---------------------------------------------------------------#
-make OPT=${OPT} KIND_COMP=${KIND} ${CLEAN} GIT_TAG=${GIT_TAG}
+#make OPT=${OPT} KIND_COMP=${KIND} ${CLEAN} GIT_TAG=${GIT_TAG}
 make_exit_code=$?
 #------------------------------------------------------------------------------------------#
 
